bin/demultiplexing.sh

```diff
@@ -0,0 +1,272 @@
+	then
+		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Runstatus compleet: ${project}."
+		##Check if it is a GAF or GD run
+		if [ ! -f "${ROOTDIR}/${GROUP}/${SCR_LFS}/Samplesheets/${project}.csv" ]
+		then
+			log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "No samplesheet for ${project}: continue."
+			continue
+		fi
+
+		### SETTING PATHS
+		### Check if the demultiplexing is already started
+		if [ ! -f "${SCR_ROOT_DIR}/logs/${project}_Demultiplexing.started" ]
+		then
+			python checkSampleSheet.py --input "${SCR_ROOT_DIR}/Samplesheets/${project}.csv" --logfile "${logFile}"
+			if [ -s "${logFile}.error" ]
+			then
+				log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "${project} skipped."
+				cat  "${logFile}.error" | mail -s "Samplesheet error ${project}" "${ONTVANGER}"
+				rm "${logFile}.error"
+				break
+			else
+				echo  "Samplesheet is OK" >> "${logFile}"
+				#####
+				## RUN PIPELINE PART ##
+				#####
+				echo "All checks are done. Logging from now on can be found: ${pipelineLogger}" >> "${logFile}"
+
+				## Check if Check file (if samplesheet is already there) is existing
+				if [ -f "${SCR_ROOT_DIR}/Samplesheets/${project}_Check.txt" ]
+				then
+					## Remove tmp Check file
+                                        rm "${SCR_ROOT_DIR}/Samplesheets/${project}_Check.txt"
+					echo "rm ${SCR_ROOT_DIR}/Samplesheets/${project}_Check.txt" >> "${pipelineLogger}"
+				fi
+
+					### Check if runfolder already exists
+				if [ ! -d "${SCR_ROOT_DIR}/generatedscripts/${project}" ]
+				then
+					mkdir -p "${SCR_ROOT_DIR}/generatedscripts/${project}/"
+					echo "mkdir -p ${SCR_ROOT_DIR}/generatedscripts/${project}/" >> "${pipelineLogger}"
+				fi
+
+				## Direct to generatedscripts folder
+				cd "${SCR_ROOT_DIR}/generatedscripts/${project}/"
+
+				## Copy generate script and samplesheet
+				cp "${SCR_ROOT_DIR}/Samplesheets/${project}.csv" "${project}.csv"
+				echo "copied ${SCR_ROOT_DIR}/Samplesheets/${project}.csv to ${project}.csv" >> "${pipelineLogger}"
+
+				cp "${EBROOTNGS_DEMULTIPLEXING}/generate_template.sh" ./
+				echo "Copied ${EBROOTNGS_DEMULTIPLEXING}/generate_template.sh to ." >> "${pipelineLogger}"
+				echo "" >> "${pipelineLogger}"
+
+
+				### Generating scripts
+                                echo "Generated scripts" >> "${pipelineLogger}"
+                                sh generate_template.sh "${project}" "${SCR_ROOT_DIR}" "${GROUP}" 2>&1 >> "${pipelineLogger}"
+
+				check=$(tail -1 "${pipelineLogger}")
+				if [[ "${check}" == *"WRONG"* ]]
+				then
+					echo "there is something wrong, EXIT"
+					echo "###"
+					echo "### Here comes the last three lines of the logger:"
+					tail -3 "${pipelineLogger}"
+					echo "###"
+					echo "###"
+					exit 1 
+				fi
+                                echo "cd ${SCR_ROOT_DIR}/runs/${project}/jobs" >> "${pipelineLogger}"
+                                cd "${SCR_ROOT_DIR}/runs/${project}/jobs"
+
+				sh submit.sh
```

Comment:
```sh``` should not be necessary when submit.sh has a proper shebang and executable perms, but if you want to prefix the interpreter ```sh``` -> ```bash```